mobile/src/common/rounded-avatar.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
```

Comment:
`common` is a shortcut for `i-do-not-know-there-to-put-my-things-folder` :). As this is an avatar, it's supposed to be somewhere in `people` folder

Reply:
I see it as a place for components what could be used elsewhere not at some specific screen only. For example, a button with a specific style what could appear in forms/screen across the app would be also placed here.